operators/aqua/2022.4.0/manifests/aqua-operator.v2022.4.0.clusterserviceversion.yaml

```diff
@@ -4,14 +4,6 @@ metadata:
   name: aqua-operator.v2022.4.0
   namespace: placeholder
   annotations:
-    # Setting olm.maxOpenShiftVersion automatically
-    # This property was added via an automatic process since it was possible to identify that this distribution uses API(s),
-    # which will be removed in the k8s version 1.22 and OpenShift version OCP 4.9. Then, it will prevent OCP users to
-    # upgrade their cluster to 4.9 before they have installed in their current clusters a version of your operator that
-    # is compatible with it. Please, ensure that your project is no longer using these API(s) and that you start to
-    # distribute solutions which is compatible with Openshift 4.9.
-    # For further information, check the README of this repository.
-    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'
```

Comment:
It is correct:

```
operator-sdk bundle validate operators/aqua/2022.4.0  --select-optional suite=operatorframework --optional-values=k8s-version=1.22
WARN[0000] Warning: Value aquasecurity.github.io/v1alpha1, Kind=AquaStarboard: provided API should have an example annotation 
WARN[0000] Warning: Value aquasecurity.github.io/v1alpha1, Kind=ConfigAuditReport: provided API should have an example annotation 
WARN[0000] Warning: Value aquasecurity.github.io/v1alpha1, Kind=ClusterConfigAuditReport: provided API should have an example annotation 
WARN[0000] Warning: Value : (aqua-operator.v2022.4.0) csv.Spec.minKubeVersion is not informed. It is recommended you provide this information. Otherwise, it would mean that your operator project can be distributed and installed in any cluster version available, which is not necessarily the case for all projects. 
WARN[0000] Warning: Value aqua-operator.v2022.4.0: unable to find the resource requests for the container: (aqua-operator). It is recommended to ensure the resource request for CPU and Memory. Be aware that for some clusters configurations it is required to specify requests or limits for those values. Otherwise, the system or quota may reject Pod creation. More info: https://master.sdk.operatorframework.io/docs/best-practices/managing-resources/ 
INFO[0000] All validation tests have completed successfully 
```